packages/blockchain-link/src/types/cardano.ts

```diff
@@ -0,0 +1,148 @@
+import { AccountInfoParams } from './params';
+
+export interface Subscribe {
```

Comment:
It's our internal lib. The same pattern is used in blockbook and ripple workers.

Please check `packages/blockchain-link/src/types/blockbook.ts` and `packages/blockchain-link/src/types/ripple.ts`

Reply:
> It's our internal lib.

And, I guess, the library isn't written in Typescript hence it doesn't expose types and hence we have to define them ourselves. It would still be cool to explain this situation in a comment. Like, for example,
```
/**
  * @blockfrost/blockfrost-js doesn't provide types for its API.
  * Define the types here manually based on https://docs.blockfrost.io/
  */
```

The link to the docs is important - we want the code in this repo to be maintainable even without an insider knowledge of some "internal lib". .)

> Please check packages/blockchain-link/src/types/blockbook.ts and packages/blockchain-link/src/types/ripple.ts

I did. Blockbook TS types are defined here  because blockchain-link *is* the de-facto client for blockbook servers. Ripple API types are actually imported from the ripple library.